packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -151,51 +72,31 @@ mv %{buildroot}%{gem_instdir}/settings.d/salt.saltfile.example \
 mv %{buildroot}%{gem_instdir}/settings.d/salt.yml.example \
    %{buildroot}%{foreman_proxy_settingsd_dir}/salt.yml
 
-mkdir -p  %{buildroot}%{salt_config_dir}
-cp -pa .%{gem_instdir}/etc/foreman.yaml.example %{buildroot}%{salt_config_dir}/foreman.yaml
```

Comment:
where do we package this and the other files?

Reply:
https://github.com/theforeman/foreman-packaging/pull/10724/files#diff-7f2b45922f25a10c95613ac59ae09ca985b02817e57bc926206b8815a454747aR128